### PR TITLE
homebrew: add Linux brew path to defaults

### PR DESCRIPTION
**Pull request by dawidd6**
##### SUMMARY

Homebrew on Linux is by default installed in `/home/linuxbrew/.linuxbrew/bin`, so let's search for brew there too.

https://docs.brew.sh/Homebrew-on-Linux#install

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

homebrew

### CODE REVIEW

**Comment by russoz** [comment]
Hi @dawidd6 
Thank you for the contribution! Please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

File: plugins/modules/packaging/os/homebrew.py [hunk 1]
```diff
@@ -876,7 +876,7 @@ def main():
                 elements='str',
             ),
             path=dict(
-                default="/usr/local/bin:/opt/homebrew/bin",
+                default="/usr/local/bin:/opt/homebrew/bin:/home/linuxbrew/.linuxbrew/bin",
                 required=False,
                 type='path',
```
Comment (russoz) [inline plugins/modules/packaging/os/homebrew.py:881]: This should/could be:
```suggestion
                type='list',
                elements='path',
```
But that would be a different PR. ;-)

**Review by russoz — COMMENTED** [review]
LGTM to me, but I have not tested in Linux.

**Review by russoz — APPROVED** [review]
LGTM (now making github happy about it)

File: changelogs/fragments/5241-homebrew-add-linux-path.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - homebrew - added Homebrew on Linux path to defaults (https://github.com/ansible-collections/community.general/pull/5241)
```
Comment (russoz) [inline changelogs/fragments/5241-homebrew-add-linux-path.yaml:2]:
```suggestion
  - homebrew - added Homebrew on Linux path to defaults (https://github.com/ansible-collections/community.general/pull/5241). 
```

**Review by russoz — APPROVED** [review]
Other than the pending adjustment in the changelog, LGTM

File: plugins/modules/packaging/os/homebrew.py [hunk 3]
```diff
@@ -39,7 +39,7 @@
             - "A C(:) separated list of paths to search for C(brew) executable.
               Since a package (I(formula) in homebrew parlance) location is prefixed relative to the actual path of I(brew) command,
               providing an alternative I(brew) path enables managing different set of packages in an alternative location in the system."
-        default: '/usr/local/bin:/opt/homebrew/bin'
+        default: '/usr/local/bin:/opt/homebrew/bin:/home/linuxbrew/.linuxbrew/bin'
```
Comment (felixfontein) [inline plugins/modules/packaging/os/homebrew.py:42]: Why not also do the same modification to the `homebrew_cask` and `homebrew_tap` modules?

Comment (dawidd6) [reply plugins/modules/packaging/os/homebrew.py:42]: Done in `homebrew_tap` too. Not done in `homebrew_cask` as Casks are not supported on Linux.

File: changelogs/fragments/5241-homebrew-add-linux-path.yaml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - homebrew, homebrew_tap - added Homebrew on Linux path to defaults (https://github.com/ansible-collections/community.general/pull/5241).
```
Comment (russoz) [inline changelogs/fragments/5241-homebrew-add-linux-path.yaml:2]: Please break it down in two lines, on for each module. The change description can be exactly the same.

Comment (felixfontein) [reply changelogs/fragments/5241-homebrew-add-linux-path.yaml:2]: I think it's also fine to have both in one line. We definitely did that before.

File: plugins/modules/packaging/os/homebrew_tap.py [hunk 5]
```diff
@@ -49,7 +49,7 @@
     path:
         description:
             - "A ':' separated list of paths to search for C(brew) executable."
```
Comment (felixfontein) [inline plugins/modules/packaging/os/homebrew_tap.py:51]: While we're at it ;)
```suggestion
            - "A C(:) separated list of paths to search for C(brew) executable."
```

**Review by felixfontein — APPROVED** [review]
LGTM from my side.

**Review by russoz — APPROVED** [review]
LGTM then :)

**Comment by felixfontein** [comment]
@dawidd6 thanks for implementing this!
@russoz thanks for reviewing!
